lib/m2b_csv.rb

```diff
@@ -0,0 +1,77 @@
+require 'csv'
+require 'faraday'
+require 'fileutils'
+
+##
+# Run this in the lib folder to get classes, properties and their bibids from a csv file
+#   with headers: className, bibid or propertyName, bibid
+# #
+class M2bCSV
+  def m2b_classes_dir
+    "../import/m2b-classes.csv".to_s
+  end
+
+  def m2b_props_dir
+    "../import/m2b-properties.csv".to_s
+  end
+
+  def output_dir
+    '../spec/fixtures'
+  end
+
+  def directory_out(m2b_dir)
+    "#{output_dir}/loc/#{m2b_dir}"
+  end
+
+  def m2b_type(obj)
+    if !obj['className'].nil?
+      obj['className']
+    elsif !obj['propertyName'].nil?
+      obj['propertyName']
+    end
+  end
+
+  def loc_sru_url
+    'http://lx2.loc.gov:210/LCDB?operation=searchRetrieve&recordSchema=marcxml&maximumRecords=1&query=rec.id'
```

Comment:
I am not suggesting we bother with it, but here is the hardcoded loc sru url ...  (also the hardcoded dir-out) -- those are the only reasons this code is loc specific, afaict.